server/utils/crmBackLink.ts

```diff
@@ -1,6 +1,10 @@
 import { Request } from 'express'
 
-function manageBackLink(currentUrl: string): string {
+function manageBackLink(currentUrl: string, lastVisitedSection: string): string {
```

Comment:
```suggestion, default lastVisitedSection param to empty string
function manageBackLink(currentUrl: string, lastVisitedSection: string = ''): string {
```
so that you can have `manageBackLink(CURRENT_URL)` instead of `manageBackLink(CURRENT_URL, '')`

Reply:
done